src/components/Tutorials/subComps/RemoveStepModal.jsx

```diff
@@ -6,88 +6,91 @@ import { useDispatch } from "react-redux";
 import { removeStep } from "../../../store/actions";
 import Snackbar from "@mui/material/Snackbar";
 import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
```

Comment:
Since we are not using this stack anymore, can you remove the import statement.

Reply:
Hi @shivareddy6 i have removed the stack import, please review.
Thanks